setup.py

```diff
@@ -47,46 +47,46 @@
 
 REQUIRED = [
     "click>=7.0.0,<8.0.2",
-    "colorama>=0.3.9",
+    "colorama>=0.3.9,<1",
     "dill==0.3.*",
-    "fastavro>=1.1.0",
-    "google-api-core>=1.23.0",
-    "googleapis-common-protos==1.52.*",
-    "grpcio>=1.34.0",
-    "grpcio-reflection>=1.34.0",
-    "Jinja2>=2.0.0",
+    "fastavro>=1.1.0,<2",
+    "google-api-core>=1.23.0,<2",
```

Comment:
We should probably do 
```suggestion
    "google-api-core>=2,<3",
```

Reply:
Yup I checked the version against 3.8 and there weren't any major version bumps. I think it's only 3.10 that has a version bump. Do you have a suggestion for what I should do. I can bump google-api-core restrictions up to 3?

Reply:
Sure, that's fine by me.